### ISSUE TITLE
Support Ubuntu 22.04

### ISSUE THREAD

**Issue by yahonda** [yahonda/multipass-rails-dev-box#2]
This issue tracks Ubuntu 22.04 support as a guest os, it is blocked by the cloud-init timeout.

```
% multipass version
multipass   1.9.0-rc.557+gc2561306.mac
multipassd  1.9.0-rc.557+gc2561306.mac
```

```
% time multipass launch 22.04 -d 20G --name rails-dev-box --cloud-init ./cloud-init.yaml
launch failed: The following errors occurred:
timed out waiting for initialization to complete
multipass launch 22.04 -d 20G --name rails-dev-box --cloud-init   0.11s user 0.19s system 0% cpu 5:22.26 total
```

```
% multipass shell rails-dev-box
Welcome to Ubuntu 22.04 LTS (GNU/Linux 5.15.0-25-generic aarch64)

 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage

  System information as of Tue Apr 26 08:34:31 AM JST 2022

  System load:             0.8359375
  Usage of /:              34.1% of 19.22GB
  Memory usage:            64%
  Swap usage:              1%
  Processes:               107
  Users logged in:         0
  IPv4 address for enp0s1: 192.168.64.40
  IPv6 address for enp0s1: fd48:fa41:3ef1:2641:5054:ff:fe58:a608

 * Super-optimized for small spaces - read how we shrank the memory
   footprint of MicroK8s to make it the smallest full K8s around.

   https://ubuntu.com/blog/microk8s-memory-optimisation

9 updates can be applied immediately.
1 of these updates is a standard security update.
To see these additional updates run: apt list --upgradable


To run a command as administrator (user "root"), use "sudo <command>".
See "man sudo_root" for details.

ubuntu@rails-dev-box:~$ cd /var/log
ubuntu@rails-dev-box:/var/log$ ls
alternatives.log  btmp                   dist-upgrade  fontconfig.log  landscape  postgresql  redis    ubuntu-advantage-timer.log
apt               cloud-init.log         dmesg         journal         lastlog    private     syslog   unattended-upgrades
auth.log          cloud-init-output.log  dpkg.log      kern.log        mysql      rabbitmq    sysstat  wtmp
ubuntu@rails-dev-box:/var/log$ cloud-init analyze show -i cloud-init.log
-- Boot Record 01 --
The total time elapsed since completing an event is printed after the "@" character.
The time the event takes is printed after the "+" character.

Starting stage: init-local
|`->no cache found @00.00200s +00.00000s
|`->found local data from DataSourceNoCloud @00.01000s +00.03700s
Finished stage: (init-local) 00.27200 seconds

Starting stage: init-network
|`->restored from cache with run check: DataSourceNoCloud [seed=/dev/vda][dsmode=net] @02.53400s +00.00200s
|`->setting up datasource @02.55400s +00.00000s
|`->reading and applying user-data @02.56500s +00.01700s
|`->reading and applying vendor-data @02.58200s +00.01200s
|`->reading and applying vendor-data2 @02.59400s +00.00000s
|`->activating datasource @02.68500s +00.00000s
|`->config-migrator ran successfully @02.68600s +00.00100s
|`->config-seed_random ran successfully @02.68700s +00.00000s
|`->config-bootcmd ran successfully @02.68700s +00.00000s
|`->config-write-files ran successfully @02.68700s +00.00100s
|`->config-growpart ran successfully @02.68800s +00.56100s
|`->config-resizefs ran successfully @03.24900s +00.04700s
|`->config-disk_setup ran successfully @03.29600s +00.00000s
|`->config-mounts ran successfully @03.29600s +00.20600s
|`->config-set_hostname ran successfully @03.50300s +00.00000s
|`->config-update_hostname ran successfully @03.50300s +00.00000s
|`->config-update_etc_hosts ran successfully @03.50300s +00.00300s
|`->config-ca-certs ran successfully @03.50600s +00.00000s
|`->config-rsyslog ran successfully @03.50600s +00.00100s
|`->config-users-groups ran successfully @03.50700s +00.03300s
|`->config-ssh ran successfully @03.54100s +00.70000s
Finished stage: (init-network) 01.71300 seconds

Starting stage: modules-config
|`->config-emit_upstart ran successfully @08.04600s +00.00000s
|`->config-snap ran successfully @08.04600s +00.00100s
|`->config-ssh-import-id ran successfully @08.04700s +00.00000s
|`->config-keyboard ran successfully @08.04700s +00.00000s
|`->config-locale ran successfully @08.04700s +00.94000s
|`->config-set-passwords ran successfully @08.98700s +00.00100s
|`->config-grub-dpkg ran successfully @08.98800s +00.08700s
|`->config-apt-pipelining ran successfully @09.07600s +00.00000s
|`->config-apt-configure ran successfully @09.07600s +24.13300s
|`->config-ubuntu-advantage ran successfully @33.20900s +00.00000s
|`->config-ntp ran successfully @33.20900s +00.00000s
|`->config-timezone ran successfully @33.20900s +00.00600s
|`->config-disable-ec2-metadata ran successfully @33.21500s +00.00100s
|`->config-runcmd ran successfully @33.21600s +00.00000s
|`->config-byobu ran successfully @33.21700s +00.00000s
Finished stage: (modules-config) 25.18800 seconds

Starting stage: modules-final
|`->config-package-update-upgrade-install ran successfully @33.39400s +270.00400s
|`->config-fan ran successfully @303.40100s +00.02000s
|`->config-landscape ran successfully @303.42200s +00.00100s
|`->config-lxd ran successfully @303.42300s +00.00000s
|`->config-ubuntu-drivers ran successfully @303.42300s +00.00100s
|`->config-write-files-deferred ran successfully @303.42400s +00.09700s
|`->config-puppet ran successfully @303.52100s +00.00100s
|`->config-chef ran successfully @303.52200s +00.00000s
|`->config-mcollective ran successfully @303.52200s +00.00000s
|`->config-salt-minion ran successfully @303.52200s +00.00100s
|`->config-reset_rmc ran successfully @303.52300s +00.00000s
|`->config-refresh_rmc_and_interface ran successfully @303.52300s +00.00000s
|`->config-rightscale_userdata ran successfully @303.52300s +00.00100s
|`->config-scripts-vendor ran successfully @303.52400s +00.00100s
|`->config-scripts-per-once ran successfully @303.52500s +00.00000s
|`->config-scripts-per-boot ran successfully @303.52500s +00.00100s
|`->config-scripts-per-instance ran successfully @303.52600s +00.00000s
|`->config-scripts-user ran successfully @303.52600s +28.56800s
|`->config-ssh-authkey-fingerprints ran successfully @332.10700s +00.04100s
|`->config-keys-to-console ran successfully @332.14900s +00.03200s
|`->config-install-hotplug ran successfully @332.18100s +00.01900s
|`->config-phone-home ran successfully @332.20000s +00.00100s
|`->config-final-message ran successfully @332.20100s +00.00400s
|`->config-power-state-change ran successfully @332.20600s +00.00100s
Finished stage: (modules-final) 298.84100 seconds

Total Time: 326.01400 seconds

1 boot records analyzed
ubuntu@rails-dev-box:/var/log$ more /etc/lsb-release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=22.04
DISTRIB_CODENAME=jammy
DISTRIB_DESCRIPTION="Ubuntu 22.04 LTS"
ubuntu@rails-dev-box:/var/log$

**Comment by yahonda** [comment]
Opened https://github.com/canonical/multipass/issues/2574

**Comment by yahonda** [comment]
Closed via #3
